### PR TITLE
gh-1495 describe option-list for socket.getaddrinfo() + fix translation

### PR DESCRIPTION
**Pull request by Onvember**

### CODE REVIEW

File: doc/reference/reference_lua/socket.rst [hunk 1]
```diff
@@ -163,14 +163,23 @@ the function invocations will look like ``sock:function_name(...)``.
...
     The ``socket.getaddrinfo()`` function is useful for finding information
     about a remote site so that the correct arguments for
     ``sock:sysconnect()`` can be passed.
     This function may use the :ref:`worker_pool_threads <cfg_basic-worker_pool_threads>`
     configuration parameter.
 
+    :param string host: URL or IP address
+    :param number port: port number or a string pointing to a port
+    :param number timeout: maximum number of seconds to wait
+    :param table options: * type -- preferred socket type
```
Comment (lenkis) [inline doc/reference/reference_lua/socket.rst:178]:
```suggestion
    :param table options: * ``type`` -- preferred socket type
```

File: doc/reference/reference_lua/socket.rst [hunk 2]
```diff
@@ -163,14 +163,23 @@ the function invocations will look like ``sock:function_name(...)``.
...
+                          * family -- desired address family for the
```
Comment (lenkis) [inline doc/reference/reference_lua/socket.rst:179]:
```suggestion
                          * ``family`` -- desired address family for the
```

File: doc/reference/reference_lua/socket.rst [hunk 3]
```diff
@@ -163,14 +163,23 @@ the function invocations will look like ``sock:function_name(...)``.
...
+                            returned addresses
+                          * protocol
```
Comment (lenkis) [inline doc/reference/reference_lua/socket.rst:181]:
```suggestion
                          * ``protocol``
```

File: doc/reference/reference_lua/socket.rst [hunk 4]
```diff
@@ -163,14 +163,23 @@ the function invocations will look like ``sock:function_name(...)``.
...
+                          * flags -- additional options (see details `here <https://man7.org/linux/man-pages/man3/getaddrinfo.3.html>`_)
```
Comment (lenkis) [inline doc/reference/reference_lua/socket.rst:182]:
```suggestion
                          * ``flags`` -- additional options (see details `here <https://man7.org/linux/man-pages/man3/getaddrinfo.3.html>`_)
```

File: doc/reference/reference_lua/socket.rst [hunk 5]
```diff
@@ -815,11 +835,11 @@ On the first shell, start Tarantool and say:
     }
     )
 
-The above code means: use `tcp_server()` to wait for a
-connection from any host on port 3302. When it happens,
-enter a loop that reads on the socket and prints what it
+The above code means: |br|
```
Comment (lenkis) [inline doc/reference/reference_lua/socket.rst:838]: Maybe we'd better present these steps as a numbered list?

File: doc/reference/reference_lua/socket.rst [hunk 6]
```diff
@@ -874,20 +894,17 @@ On the first shell, start Tarantool and say:
...
 
-The above code means: use ``tcp_server()`` to wait for a
-connection from any host on port 3302.
-Specify that there will be an initial call to ``prepare``
-which displays something about the server,
-then calls ``setsockopt(...'SO_REUSEADDR'...)`` (this is the same
-option that Tarantool would set if there was no
-``prepare``),
-and then returns 5 (this is a rather low backlog queue size).
+The above code means: |br|
```
Comment (lenkis) [inline doc/reference/reference_lua/socket.rst:897]: Same suggestion about a numbered list.

**Review by lenkis — CHANGES_REQUESTED** [review]

File: doc/reference/reference_lua/socket.rst (hunk 6 above)
Comment (Onvember) [reply doc/reference/reference_lua/socket.rst]: fixed in e64c567

File: doc/reference/reference_lua/socket.rst (hunk 5 above)
Comment (Onvember) [reply doc/reference/reference_lua/socket.rst]: fixed in e64c567

**Pull request merged**